src/main/kotlin/jambda/camp/clean/domain/Customer.kt

```diff
@@ -0,0 +1,12 @@
+package jambda.camp.clean.domain
+
+data class Customer(
+    override val id: UserId,
+    val name: CustomerName,
+    val shoppingCart: ShoppingCart
+): User(id)
+
+class CustomerName (
+    val firstName: String,
```

Comment:
Was haltet ihr von Domain Primitives (DDD)?

Bei den anderen Klassen wurde diese schon ansatzweise so implementiert.
Da würde nur noch eine Factory Methode zur Instanzierung und die Validierung fehlen.

Vorschlag für den ersten weiteren Schritt in diese Richtung:
```
data class CustomerFirstName(
 val firstName: String
)

data class CustomerLastName(
 val lastName: String
)
```